doc-content/enterprise-only/project-data/migrating-legacy-designer-proc.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Migrating business processes to the new process designer
 
-The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. If you intend to use the new process designer, start migrating your processes to the new designer. Create all new processes in the new process designer.
+The legacy process designer in {CENTRAL} is deprecated in {PRODUCT_PAM} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. If you intend to use the new process designer, start migrating your processes to the new designer. Create all new processes in the new process designer.
```

Comment:
I would keep PRODUCT, otherwise please replace all occurences in the paragraph.

Reply:
Done